api-report/purchases-js.api.md

```diff
@@ -107,7 +107,7 @@ export enum ErrorCode {
 
 // @public
 export interface HttpConfig {
-    includeCredentials?: boolean;

```

Comment:
I removed the `includeCredentials` parameter since we're not sure we want to include it. For now, we will just add the `additionalHeaders` parameter here.